Core/Model/User.php

```diff
@@ -265,12 +265,6 @@ public function verifyPassword($value)
             return true;
         }
 
-        // To ensure that any user of facturascripts_2015 can login and rehash its password
```

Comment:
We need this code.

Reply:
ok, I do not know why that change in my branch comes out. Please apply the change and I will update my master branch with the correct code.